maestro-cli/src/main/java/maestro/cli/command/TestCommand.kt

```diff
@@ -97,7 +107,11 @@ class TestCommand : Callable<Int> {
             throw CliError("--platform option was deprecated. You can remove it to run your test.")
         }
 
-        return MaestroSessionManager.newSession(parent?.host, parent?.port, parent?.deviceId) { session ->
+        val launchWebDevice = isWebFlow().also {
+            if (it) println("WARNING! Web support is an experimental feature and may be removed in future versions.\n")
```

Comment:
nit: use `PrintUtils.warn()` instead of `println`. It adds warning color to the message